README.md

```diff
@@ -197,7 +197,15 @@ You can use one or more of the following magic sequence which will get substitut
 - `authorEmail`: E-mail of video publisher
 - `uniqueId`: An _unique-enough_ ID generated from the video metadata
 
-Example -
+Examples -
+```
+Input:
+    -t 'This is an example'
+
+Expected filename:
+    This is an example.mkv
+```
```

Comment:
Please merge this code brackets with the other example, add another mixed example like 

-t 'This is an example by {author}'